packages/interbit/src/file/packCovenants.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs-extra')
+const promisify = require('util').promisify
+const exec = promisify(require('child_process').exec)
+const path = require('path')
+const {
+  constants: { ROOT_CHAIN_ALIAS }
+} = require('interbit-covenant-tools')
+
+const { startInterbit } = require('../chainManagement')
+
+// Note: Because interbit-core does not expose it's hashing function (yet) we ned to
+// start a cli and deploy the covenants to get a valid hash that is in sync with interbit
+// TODO: update this to use the hashing function instead of starting a CLI -- Blocked until core releases hashing algo in API
+const packCovenants = async (location, covenantConfig) => {
```

Comment:
Is there an issue for exposing the hashing algorithm?

Reply:
Yes!